### PR TITLE
Improve CI (Aarch64, Power)

### PR DESCRIPTION
**Pull request by Mizux**
Few updates

* cleanup scripts
* Add Aarch64 bootlin support (script, cmake/ci, workflows)
* Add Power bootlin support (script, cmake/ci, workflows)

### CODE REVIEW

File: scripts/run_integration.sh [hunk 1]
```diff
@@ -145,6 +141,27 @@ function expand_bootlin_config() {
...
+    "ppc64le")
+      local -r POWER_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/powerpc64le-power8/tarballs/powerpc64le-power8--glibc--stable-2021.11-1.tar.bz2"
+      local -r GCC_PREFIX="powerpc64le"
+      ;;
+    "ppc64")
+      local -r POWER_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/powerpc64-power8/tarballs/powerpc64-power8--glibc--stable-2021.11-1.tar.bz2"
+      local -r GCC_PREFIX="powerpc64"
+      ;;
+    "ppc")
+      #local -r POWER_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/powerpc-e500mc/tarballs/powerpc-e500mc--glibc--stable-2021.11-1.tar.bz2"
```
Comment (gchatelet) [inline scripts/run_integration.sh:161]: Why is it commented out?

File: scripts/run_integration.sh [hunk 2]
```diff
@@ -230,12 +253,14 @@ function expand_codescape_config() {
     "mips64")
       MIPS_FLAGS="-EB -mips64r6 -mabi=64"
       FLAVOUR="mips-r6-hard"
+      #MIPS_FLAGS="-EB -mips64r2 -mabi=64"
```
Comment (gchatelet) [inline scripts/run_integration.sh:256]: same here and below, why do you need to keep these comments?

File: scripts/run_integration.sh (hunk 1 above)
Comment (Mizux) [reply scripts/run_integration.sh:161]: While QEmu provide a single `ppc` target (as well as docker), bootlin provide several flavors
`440fp` and `e500mc` for PowerPC while here I arbitrarily the `440fp` we could have choose the other one...

An other way would be to have `ppc-e500mc` and `ppc-440fp` (and `ppc` alias of `ppc-440fp` ?)

ref:
```sh
docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
...
Setting /usr/bin/qemu-ppc-static as binfmt interpreter for ppc
Setting /usr/bin/qemu-ppc64-static as binfmt interpreter for ppc64
Setting /usr/bin/qemu-ppc64le-static as binfmt interpreter for ppc64le
```

note: same issue with `ppc64[le]`: `power8`, `e5500`, 'e6500', `le-power8`...

File: scripts/run_integration.sh (hunk 2 above)
Comment (Mizux) [reply scripts/run_integration.sh:256]: same here maybe we could have a flag to decide which version to use so user can easily reuse this script to target the correct CPU by just changing one line, so while we only enable one flavor it's easy for the community to adapt.

Or again we should/could have both `mips-r6`/`mips-r2` and a `mips` alias to target the last one aka `mips-r6` ?

**Pull request merged**